apps/condo/domains/common/utils/fileAdapter.js

```diff
@@ -18,10 +18,7 @@ class FileAdapter {
             case 'sbercloud':
                 Adapter = this.createSbercloudFileApapter()
                 if (!Adapter) {
-                    // no fallback for production 
```

Comment:
We can fail without config at the moment

Reply:
Done